offsite-activity-signup/src/appsscript.json

```diff
@@ -0,0 +1,4 @@
+{
+  "timeZone": "America/Denver",
```

Comment:
Was debating what to do about the timezone for my own solution. Not sure it makes a big difference either way.

Reply:
Whatever we we put will be wrong for somebody :)